setup.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2017, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+from setuptools import setup
+from setuptools import find_packages
+
+setup(
+    name='flask_saml_sso',
```

Comment:
This name sort of indicates that this is a generally useful module, for anyone wanting to do SAML SSO with Flask. Is that the case?

Reply:
Yes. Sort of. With the added functionality that we also create a SQL alchemy session interface.